### ISSUE TITLE
DAX Editor doesn't seem to support certain syntax 

### ISSUE THREAD

**Issue by ay5h** [DaxEditor/DaxEditor#11]
DAX Editor doesn't seem to support the following two syntax although they work fine in the model. 

Sample script that won't save:
 
CREATE MEASURE 'Sales'[Calendar Sales Amount] = SUM(Sales[USD_Sales_Amount])
CALCULATION PROPERTY GENERAL
    FORMAT = '#,0'
;
 
CREATE MEASURE 'Sales'[No of Rows] = COUNTROWS(Sales)
CALCULATION PROPERTY GENERAL
    FORMAT = '#,0'
;
 
CREATE MEASURE 'Sales'[Calendar CAGR Ratio] = IF( ISBLANK( CALCULATE ([Calendar Sales Amount], PARALLELPERIOD('Period'[Date], -5, year)) )
   , BLANK()
   , DIVIDE ( [Calendar Sales Amount] , CALCULATE ([Calendar Sales Amount], PARALLELPERIOD('Period'[Date], -5, year)) )
)
CALCULATION PROPERTY GENERAL
    VISIBLE = False
;
 
CREATE MEASURE 'Sales'[Calendar Sales CAGR] =  IF( ISBLANK([Calendar CAGR Ratio]), BLANK()
   , ( ([Calendar CAGR Ratio]^0.2) -1 ) * 100 )
CALCULATION PROPERTY GENERAL
    FORMAT = '0.00'
;
 
CREATE MEASURE 'Sales'[Model Last Processed]] =  MAX(Currency[Processed Date])+(5/24)
;

**Comment by marcosqlbi** [comment]
This is a bug and we should fix it, supporting the "^" operator:
([Calendar CAGR Ratio]^0.2) 

This statement has a double square bracket that is invalid, and Currency is a reserved keyword:
CREATE MEASURE 'Sales'[Model Last Processed]] = MAX(Currency[Processed Date])+(5/24)

Probably Currency should be accepted, but it would be better to rename the table name to 'Currency'
But the double square bracket seems an error in the code - can you check it again?
Thanks,

Marco

**Comment by ay5h** [comment]
Yes Currency without single quotes definitely doesn't work in the editor although does work in the solution and gives the correct timestamp result. If I include the quotes then yes, that specific measure does save successfully.
Thanks!
